src/storage/src/mem_table.rs

```diff
@@ -126,7 +141,9 @@ impl MemTable {
                             new: KeyOp::Delete(old_value),
                         }));
                     }
+                    self.kv_size.add(e.key(), &old_value);
```

Comment:
Did we miss to subtract the original_value size here?

Reply:
Yes, fixed.